frontend/src/components/MapView/Legends/LegendItem/__snapshots__/index.test.tsx.snap

```diff
@@ -104,22 +104,21 @@ exports[`renders as expected 1`] = `
               />
               <span
                 class="MuiSlider-track"
-                style="left: 0%; width: 50%;"
+                style="left: 0%;"
               />
               <input
                 type="hidden"
-                value="0.5"
+                value="NaN"
```

Comment:
is this expected @echaidemenos ?

Reply:
Yes, initial value of the state is undefined, so the components know it is the first time referenced and they have to set it. That way we keep the value when toggling layers on and off